.github/workflows/gh-ci.yaml

```diff
@@ -119,7 +119,7 @@ jobs:
         # software (curl, ssh, ...), but the version in biobuilds are very
         # outdated. This can lead to problem when loading shared libraries,
         # see https://github.com/MDAnalysis/mdanalysis/pull/3126#issuecomment-813112080
-        channels: conda-forge, biobuilds
+        channels: conda-forge, bioconda
```

Comment:
As far as I'm aware biobuild still works, but it hasn't been updated since 2019? (https://github.com/biobuilds/biobuilds), if purely on a security basis, it probably is better to just use the better maintained bioconda here?

Reply:
I don't think that there's anything that ties us to biobuilds.